meta-oe/recipes-support/driverctl/driverctl_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "driverctl is a device driver control utility for Linux"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=1a6d268fd218675ffea8be556788b780"
+
+DEPENDS = "systemd"
```

Comment:
is this hard dependency ? if so then make sure it check for systemd in DISTRO_FEATURES as required distro features.

Reply:
you might want to inherit systemd instead and see if that works.